packages/ui-kit/TextViewer/TextViewer.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+import cn from 'classnames';
+
+import { TextViewerVariant } from './types';
+
+import css from './TextViewer.module.scss';
+
+export interface TextViewerProps {
+    text: string;
+    variant?: TextViewerVariant;
+    title?: string;
+    subText?: string;
+}
+
+const TextViewer: FC<TextViewerProps> = ({
+    title,
+    variant = TextViewerVariant.PRIMARY,
+    text,
+    subText,
+}) => {
+    return (
+        <div className={css.block}>
+            <div className={cn(css.blockcontent, css[variant])}>
```

Comment:
модульные стили юзаем
```suggestion
            <div className={cn(css.content, css[variant])}>
```
так и понятнее